### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by anasanjuan**

### CODE REVIEW

**Comment by manuelbarzi** [comment]
move files to cv-styled folder

File: staff/ana-sanjuan/cv/styles.css [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+header {
+    text-align: right;
+}
+
+.introduction {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    margin-left:auto;
```
Comment (Arcoders) [inline staff/ana-sanjuan/cv/styles.css]: margin-left: auto;
margin-right: auto;

try it with  margin: 0 auto;

File: staff/ana-sanjuan/cv/index.html [hunk 2]
```diff
@@ -5,121 +5,115 @@
...
+            </div>
+            <div class='section__text'>
+                <p>Hello and Wellcome</p>
+                <h2 class='section__title'>I'm ANA SANJUÁN</h2>
+                <address>
+                    <p>Email: <a href= "mailto:lorem@lorem.com">lorem@lorem.com</a></p>
+                    <p>Address: Lorem Ipsum Street 1, Barcelona<br></p>
+                    <p>Phone: +00-5555-555</p>   
+                </address>
+                <nav>
```
Comment (JudithGo) [inline staff/ana-sanjuan/cv/index.html]: Is necessary the tag nav? Is it a navigation menu? 
Anyway, I think the nav tag includes a list ul. -->  https://www.w3schools.com/css/css_navbar.asp

File: staff/ana-sanjuan/cv/styles.css (hunk 1 above)
Comment (anasanjuan) [reply staff/ana-sanjuan/cv/styles.css]: good tip! thanks man! :)

File: staff/ana-sanjuan/cv/index.html (hunk 2 above)
Comment (anasanjuan) [reply staff/ana-sanjuan/cv/index.html]: thanks for the comment. 
I checked the FM and I see now that nav is only for navigation inside your page, I didn´t know it before! :)
Also, as you suggested, I will use a list here

File: staff/ana-sanjuan/cv/index.html [hunk 3]
```diff
@@ -5,121 +5,115 @@
...
+            <section class='hobbies section'>
+                <h2 class='section__title'>Hobbies</h2>
+                
+                <ul class='section__ulist star'>
+                    <li>Lindy Hop 
+                    <span class="fa fa-star checked"></span>
+                    <span class="fa fa-star checked"></span>
+                    <span class="fa fa-star checked"></span>
+                    <span class="fa fa-star"></span>
+                    <span class="fa fa-star"></span></li>
```
Comment (LluisBagur) [inline staff/ana-sanjuan/cv/index.html:88]: in this case, why do you use span? the classes used in the span are not in the css file.

Comment (anasanjuan) [reply staff/ana-sanjuan/cv/index.html:88]: fa and fa-star are the clasess that define the star per se, and checked is the filler.
fa and fa-star are taken from the link("https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"), and checked is in the css

**Pull request merged**